doc/crypto/CTLOG_STORE_get0_log_by_id.pod

```diff
@@ -0,0 +1,45 @@
+=pod
+
+=head1 NAME
+
+CTLOG_STORE_get0_log_by_id -
+Get a Certificate Transparency log from a CTLOG_STORE
+
+=head1 SYNOPSIS
+
+ #include <openssl/ct.h>
+
+ const CTLOG *CTLOG_STORE_get0_log_by_id(const CTLOG_STORE *store,
+                                         const uint8_t *log_id,
+                                         size_t log_id_len);
+
+=head1 DESCRIPTION
+
+A Signed Certificate Timestamp (SCT) identifies the Certificate Transparency
+(CT) log that issued it using the log's LogID (see RFC 6962, Section 3.2).
+Therefore, it is useful to be able to look up more information about a log
+(e.g. its public key) using this LogID.
+
+B<CTLOG_STORE_get0_log_by_id>() provides a way to do this. It will find a CTLOG
```

Comment:
Thanks for the tip, I wasn't aware of that tool. I've resolved all of the issues it highlighted.